sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt

```diff
@@ -0,0 +1,7 @@
+[adaptation-community-native-common]
+repos = adaptation-community-native-common
+pattern = Feature adaptation community native common
+description = Common packages needed by community's native HW adaptation
+
+[repositories-release]
+adaptation-community-native-common = http://repo.merproject.org/obs/nemo:/@FLAVOUR@:/hw:/native-common/sailfish_latest_%(arch)/ 
```

Comment:
Perhaps I'm missing something, but how should we handle `@FLAVOUR@`? I think that the best course of action would be moving this ssu feature to the `community-adaptation` srcpackage, creating new packages `community-adaptation-native-common-{devel,testing,localbuild}` and then make droid-configs depend on the virtual package providing it.

That said, I'm fine with hardcoding `devel` for now.

@piggz @mlehtima thoughts?

Reply:
lets go with devel for now, as thats the only options anyway?